gateway/values.yaml

```diff
@@ -144,7 +144,10 @@ appSecrets:
       <dkim-private-key>
   abac:
     oidcClientSecret: <oidc-client-secret> # The client secret used when authenticating against platform services.
-  
+  oauth2:
```

Comment:
maybe call this `xoauth2`? `oauth2` is pretty general and there is nothing to indicate that this is a sasl mechanism.